new-readme.md

```diff
@@ -3,30 +3,31 @@
 
-reSolve is a framework for developing apps based on **CQRS**, **Domain-Driven Design** and **Event Sourcing** using **React + Redux** on the client. It negates differences between your domain and technical languages, and allows you to focus on application code.
+A JavaScript framework for **CQRS**, **Domain-Driven Design** and **Event Sourcing** with good support for **React** and **Redux**. **reSolve** provides a new way to describe app's core logic and fully separate it from an auxiliary code.
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-technologies.png"></p>
 
-* **2 minute** easy [setup](#installation)
-* **Free and open source.** reSolve is free to use under the [MIT license](https://github.com/reimagined/resolve/blob/master/LICENSE.md).
-* **Flexibility.** Great opportunities with [clear API](#documentation).
+* 2 minutes [installation](#installation).
+* [Free and open source](https://github.com/reimagined/resolve/blob/master/LICENSE.md).
+* Flexible and clear [API](#documentation).
 
+<!-- TODO: text="Try it now!" -->
 [<p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-start-button.png"></p>](#getting-started)
 
 ## **📑 Table of Contents**
-* #### ️️️[🏗️ Why reSolve](#why-resolve)
-* #### [🚀 Getting Started](#getting-started)
+* **[🏗️ Why reSolve](#why-resolve)**
+* **[🚀 Getting Started](#getting-started)**
     * [Installation](#installation)
     * [Tutorials](#tutorials)
     * [Examples](#examples)
-* #### [📚 Documentation](#documentation)
+* **[📚 Documentation](#documentation)**
     * [Compatibility](#compatibility)
     * [Bacic Concepts](#basic-concepts)
-* #### [📢 Get in Touch](#get-in-touch)
+* **[📢 Get in Touch](#get-in-touch)**
 
 ## <a name="why-resolve">**🏗️ Why reSolve**</a>
 
-With reSolve you can easy and fast create different level difficulty applications. It could be applications like simple [ToDo List](#todo-list), or some reactive applications like StackOverflow, with distributed, highload and serverless architecture. Thanks to stack of technologies (CQRS, Event Sourcing, Domain-Driven Design, React + Redux) your application would be easy scalable, maintainable, so you can focus on application essence.
+With **reSolve** you can create applications of any complexity level more efficiently. A single approach is used for simple [ToDo List](#todo-list) and for a complex reactive application like [Hacker News](https://github.com/reimagined/hacker-news-resolve). The technologies stack (CQRS, Event Sourcing, Domain-Driven Design, React + Redux) makes any **reSolve** application scalable and maintainable, so you can focus on business logic instead of an auxiliary code.
```

Comment:
Hacker News is not complex reactive application

Reply:
I'll change this, but it's highly recommended to add the SO app example...